dev/Installer/src/install.cpp

```diff
@@ -0,0 +1,194 @@
+﻿#include "pch.h"
+#include "packages.h"
+#include "install.h"
+
+using namespace winrt;
+using namespace Windows::ApplicationModel;
+using namespace Windows::Foundation;
+using namespace Windows::Management::Deployment;
+using namespace Windows::System;
+
+namespace ProjectReunionInstaller {
+
+    winrt::hresult AddPackage(const Uri& packageUri)
+    {
+        PackageManager packageManager;
+        const auto deploymentOperation{ packageManager.AddPackageAsync(packageUri, nullptr, DeploymentOptions::None) };
+        deploymentOperation.get();
+        if (deploymentOperation.Status() != AsyncStatus::Completed)
```

Comment:
I _think_ this `if` block is no longer necessary - the deployment operation if it fails will cause a throw during the 'get' so this `Status` should always be `Completed` at this point.  Maybe ask Carl?

(Applies to all similar blocks.)

Reply:
It is necessary because the operation doesn't throw when it fails, it returns a failure status. This is observed in the Provisioning operation, which regularly fails with access denied whenever called by a non-admin, nor not supported when called on a framework. These failures don't result in an exception, just a failed operation.